src/output.ts

```diff
@@ -17,7 +17,7 @@ const writeFile = promisify(fs.writeFile);
 const mkdir = promisify(fs.mkdir);
 
 export async function outputReadme(readmeFilePath: string, data: DocsData) {
-  if (typeof readmeFilePath !== 'string') {
```

Comment:
It's amazing webstorm is recommending this. If this were a library targeting JS users, this would be a totally valid check.

Reply:
WebStorm is absolutely amazing in a million different ways. Anyone not using it doesn't know what they're missing. 😁

> If this were a library targeting JS users, this would be a totally valid check.

Not quite. If it were targeting JS users, the type would have been `unknown` or something like that. I checked all of these changes to be sure that nothing other than strings could be passed in even inadvertently.